x-pack/plugins/apm/public/components/shared/charts/ErrorRateChart/index.tsx

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { EuiTitle } from '@elastic/eui';
+import theme from '@elastic/eui/dist/eui_theme_light.json';
+import { i18n } from '@kbn/i18n';
+import mean from 'lodash.mean';
+import React, { useCallback, useMemo } from 'react';
+import { useChartsSync } from '../../../../hooks/useChartsSync';
+import { useFetcher } from '../../../../hooks/useFetcher';
+import { useUrlParams } from '../../../../hooks/useUrlParams';
+import { callApmApi } from '../../../../services/rest/createCallApmApi';
+import { unit } from '../../../../style/variables';
+import { asPercent } from '../../../../utils/formatters';
+// @ts-ignore
+import CustomPlot from '../CustomPlot';
+
+const tickFormatY = (y?: number) => {
+  return asPercent(y || 0, 1);
+};
+
+export const ErrorRateChart = () => {
+  const { urlParams, uiFilters } = useUrlParams();
+  const syncedChartsProps = useChartsSync();
+
+  const { serviceName, start, end, errorGroupId } = urlParams;
+  const { data: errorRateData = [] } = useFetcher(() => {
+    if (serviceName && start && end) {
+      return callApmApi({
+        pathname: '/api/apm/services/{serviceName}/errors/rate',
+        params: {
+          path: {
+            serviceName,
+          },
+          query: {
+            start,
+            end,
+            uiFilters: JSON.stringify(uiFilters),
+            groupId: errorGroupId,
+          },
+        },
+      });
+    }
+  }, [serviceName, start, end, uiFilters, errorGroupId]);
+
+  const average = useMemo(() => {
+    return tickFormatY(mean(errorRateData.map((rate) => rate.y)));
+  }, [errorRateData]);
+
+  const combinedOnHover = useCallback(
+    (hoverX: number) => {
+      return syncedChartsProps.onHover(hoverX);
+    },
+    [syncedChartsProps]
+  );
+
+  return (
+    <>
+      <EuiTitle size="xs">
+        <span>
+          {i18n.translate('xpack.apm.errorRateChart.title', {
+            defaultMessage: 'Error Rate',
+          })}
+        </span>
+      </EuiTitle>
+      <CustomPlot
```

Comment:
I'm seeing two legends in the tooltip? 
<img width="479" alt="Screenshot 2020-06-04 at 21 03 40" src="https://user-images.githubusercontent.com/4104278/83800132-3bb9c100-a6a7-11ea-9ad8-077530ccfaa6.png">